articles/tensorflow-vs-pytorch/index.md

```diff
@@ -0,0 +1,113 @@
+
+### What is PyTorch?
+
+PyTorch is a relatively new deep learning framework as it came out a year later after TensorFlow was released. The name PyTorch was inspired by a popular "[torch](http://torch.ch/)" deep learning framework, which was written in the [Lua](https://www.lua.org/) programming language. However, learning the Lua programming language was a big barrier, especially when learning deep learning. It also didn't offer the modularity necessary to interface with other deep learning libraries currently present. 
+
+This problem was solved by a team of AI researchers inspired by the torch's programming style. They decided to develop an improvised form of torch with the Python (py) programming language, which later became known as PyTorch. 
+
+Some notable features of PyTorch include:
+
+1. It is extremely fast and feels Native.
+2. It has native support for Python and all of its libraries. This feature makes it very easy to learn. 
+3. It's easy to use API that ensures easy usability when making use of the API.
+4. It is actively maintained and used by the developers at Facebook.
+5. It supports Nvidia's [CUDA](https://en.wikipedia.org/wiki/CUDA), ensuring that the code runs on a GPU, decreasing the time needed to run the code.
+6. Dynamic Computation Graphs is a key feature in PyTorch. It ensures that the graphs build up dynamically as you code.
+
+Let's begin by discussing some of the comparisons between TensorFlow and PyTorch. 
+
+### Computation Graphs
+
+From the start, the computation graph was a notable difference between TensorFlow and PyTorch.
+
+From its onset, PyTorch was built around the concept of dynamism. It provided developers with the ability to dynamically define and manipulate graphs at runtime while it is executing. This feature offered PyTorch developers an advantage over TensorFlow as manipulating graphs during runtime helped easily troubleshoot the model in case of any issue with the code.
+
+By default, when [TensorFlow](https://www.tensorflow.org/) was developed, it let developers create static computation graphs at compile time. It didn't allow us to define and manipulate graphs at runtime. If we wanted to make any changes, we would have to rebuild the neural network from the start. 
+
+But, with the release of TensorFlow 2.0, they introduced the "eager mode." They aimed to allow graphs to be defined and manipulated dynamically in computation while executing as was with PyTorch, which already had this feature, which worked so well. Rather than waiting until everything is coded up and connected before you can see if anything is done right, you could now "eagerly" execute your operations as you're working. 
+
+Today, both frameworks allow both the static and dynamic computation of graphs. No much difference between the two as far as computation graphs are concerned. 
```

Comment:
Rewrite the last sentence. As far as computational graphs are concerned, not much difference exists between the two.